modules/vulnerability-management20-download-reports.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_content-type: PROCEDURE
+[id="vulnerability-management20-download-reports_{context}"]
+= Downloading vulnerability reports
+
+[role="_abstract"]
+You can generate an on-demand vulnerability report and then download it.
++
+[NOTE]
+====
+You can only download reports that you have generated; you cannot download reports generated by other users.
+====
+
+.Procedure
+. In the {product-title-short} web portal, navigate to *Vulnerability Management 2.0* -> *Vulnerability Reporting* and locate the report you want to download in the list of reports. Generate the vulnerability report by using one of the following methods:
```

Comment:
"...and locate the report configuration for which you want to generate a downloadable report in the list of report configs"